index.html

```diff
@@ -29,38 +29,38 @@
 </head>
 
 <body>
-    <nav class="navbar navbar-expand-lg navbar-dark fixed-top mb-5 bg-black shadow-sm" id="header"
-        aria-label="Main navigation">
+    <nav class="navbar navbar-expand-xxl navbar-dark bg-black sticky-top" aria-label="Seventh navbar example">
```

Comment:
In the future, pelase no navbar-expand-xxl ... that's wayyyyy to wide of a navigation bar :)